pkg/runtime/runtime.go

```diff
@@ -1,13 +1,48 @@
-/*
-  Copyright (c) 2018, Sylabs, Inc. All rights reserved.
-
-  This software is licensed under a 3-clause BSD license.  Please
-  consult LICENSE file distributed with the sources of this project regarding
-  your rights to use or distribute this software.
-*/
-
-/*
-Package runtime holds the container runtime
-process logic
-*/
 package runtime
+
+import (
+	"net"
+
+	specs "github.com/opencontainers/runtime-spec/specs-go"
+	config "github.com/singularityware/singularity/pkg/runtime/config"
+)
+
+// Generic runtime engine
+type RuntimeEngine struct {
+	*config.RuntimeConfig
+	Runtime
+}
+
+type RuntimeCLI struct {
```

Comment:
Is this supposed to be `type RuntimeOCI struct` by any chance?

Reply:
@cclerget I think you have a typo then ;) This reads `RuntimeCLI` instead of `RuntimeOCI`

Reply:
@bauerm97 not a typo interface, RuntimeCLI struct contains OciRuntime interface for CLI side. I need to do the same RuntimeEngine